### ISSUE TITLE
Manual Checking of Scripticus Quests Force Checks Other Unchecked Quests

### ISSUE THREAD

**Issue by Clementiney** [Idleon-Companion/IdleonCompanion#223]
# Bug Report

**Description**

I'm trying to manually track a Mage character's Scripticus quest progression up to Champion of the Grasslands. I am finding this to be impossible because upon checking off Hardcore Gamer Status, Here I Come!, the bottom four quests "What Lies at the Heart of the Forest, The Bigger They Are, The Bigger They Fall!, Champion of the Grasslands, Can't Somebody Else Do It?" automatically complete as well. If I try to uncheck them, they uncheck "Hardcore Gamer Status, Here I Come!," as well.

If I try to check another quest before those four quests, the same thing happens. If I uncheck them after they are checked, they uncheck any other quest I've clicked.

![Auto Checking Quests](https://user-images.githubusercontent.com/40348268/161441966-c34144e5-42ca-4c5a-8f99-e917580fcac7.png)

---

**Details**

OS (e.g. Windows, iOS): Windows 10 Pro 64 Bit
Browser (e.g. Chrome, Safari): Brave [Version 1.37.109 Chromium: 100.0.4896.60 (Official Build) (64-bit)](https://brave.com/latest/)
Idleon Companion Version (see changelog): 1.0.1 (Stamps & Alchemy Bugfixes) | October 2021

**Comment by Clementiney** [comment]
I'm just realizing that these quests are linked and this probably isn't a bug. Woops.

**Comment by Jutakl** [comment]
Nope. Upon further reflection, "Champion of the Grasslands" cannot remain unchecked while checking off all of the other quests I've done on a character. Scripticus' quest tracking on the site is broken.
